src/smokeTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/PayHubApiHealthTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor;
+
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import static org.hamcrest.Matchers.is;
+
+@TestPropertySource("classpath:application.conf")
+class PayHubApiHealthTest {
+
+    private static final String TEST_PAY_HUB_URL = ConfigFactory.load().getString("test-pay-hub-url");
```

Comment:
we should check health of the service what we're deploying

Reply:
Deploy stage in pipeline will do the health check for app ?

Reply:
remove it then

Reply:
well... if this apps health check verifies payhub